code/src/test/test__sbi_probe_extension/test__sbi_probe_extension.c

```diff
@@ -0,0 +1,46 @@
+#include "test_macros.h"
+
+int test_case(void);
+
+/**
+ * @brief interrupt handler function that runs in S-mode. Not used in this test
+ * 
+ */
+__attribute__((interrupt("supervisor")))
+void s_mode_trap(void) {
+  uint64_t cause = read_csr(scause);
+
+  switch (cause) {
+    default:
+      break;
+  }
+}
+
+/**
+ * @brief Test case example checks for illegal instruction exception
+ * 
+ * @return int 
+ */
+int test_case(void) {
+  struct sbiret get_spec = sbi_probe_extension(EID_10);
+  if (get_spec.value == 1) {
+    if (get_spec.error == SBI_SUCCESS) {
+        exit_test(TEST_PASS);}
+  } else {
+    exit_test(TEST_FAIL);}
+
+  struct array {
+        uint64_t a[21];
+  } myArray = { { 0x01ul, 0x02ul, 0x03ul, 0x04ul, 0x05ul, 0x06ul, 0x07ul, 0x08ul,
```

Comment:
Please add a description for these values, it would be better to define them in an enum instead of an array, so that the variables are named. You can start with names like `eid__sbi_timer` or `eid__xyz`

Reply:
Looks good otherwise